### DISCUSSION TITLE
LTR

### DISCUSSION THREAD

**Discussion by t-smirnov** [meilisearch/product#592] [Feedback & Feature Proposal]
What do you think if it's possible to make Learning To Rank feature available? Would be popular in e-commerce as a killer feature.

**Comment by gmourier** [comment]
Hello @t-smirnov 👋

For now, Meilisearch does not supports any ML technics internally and we are not yet planning to do so in the short term.

Could you explain why do you think LTR would be a killer feature for e-commerce?

**Comment by t-smirnov** [comment]
it wouldn't be killer feature for small businesses, but for the medium and big ones, with 1k+ Search RPS it would be :) Previously I had experience in a top 1 Russian e-commerce and it was a good challenge with ES + LTR which brought good significant boost in CR regarding the numbers.
